### ISSUE TITLE
Type declarations are not published to npm registry

### ISSUE THREAD

**Issue by kuskoman** [lavalibs/discord.js#1]
The library is not emitting type declarations so it's kind of useless when used with TypeScript in strict mode

**Comment by appellation** [comment]
Good catch! You're welcome to open a PR for a fix or we'll get to it eventually.

**Comment by kuskoman** [comment]
As I guess it's caused by gulp, which I think is useless in these times since you are able to build files just using typescripts `tsc` command.
If you agree with me I will prepare PR replacing gulp with typescript compiler and modify its (typescripts) config

**Comment by appellation** [comment]
Sounds great. I don't recall why we used Gulp, but using plain TSC is definitely preferred here for such a simple build process.

**Comment by DevYukine** [comment]
Hey thanks for the information but you are kinda wrong, its not that i dont emit type declarations but somehow npm didn't upload them, locally i have them as expected so the PR (#2) is also kinda pointless to me, i will release a new minor later today with type declarations as expected ^^.

For why i used Gulp is just convenience so i dont mind you removing/replacing it.
